### DISCUSSION TITLE
Present select options natively on mobile

### DISCUSSION THREAD

**Discussion by javadev-jef** [csandman/chakra-react-select#254] [Q&A] [answered]
Hello, I'm looking for how to display select options on mobile natively, but I haven't found any information. Could you let me know if there is such a possibility?

Expected example:
![image](https://user-images.githubusercontent.com/62752729/230692732-a3d93142-58cc-4750-9f4a-2df6b3413759.png)

**Comment by csandman** [comment, marked as answer]
No, there's nothing built in that would allow you to offer a native select on mobile and Chakra React Select on desktop. I actually have thought about that use case before, as I know the built in select UI can be a better experience on mobile, but this is not something within the scope of this project to extend from the base package, React Select.

You could do something like this on your own, using a package like [`react-device-detect`](https://www.npmjs.com/package/react-device-detect) to determine if the user is on a mobile device and conditionally rendering one or the other, but you'd have to do it mostly manually.

**Reply by javadev-jef** [reply]
Understood. Thanks!
